### ISSUE TITLE
Add color option to media player card

### ISSUE THREAD

**Issue by pandaboy6621** [piitaya/lovelace-mushroom#328]

**Comment by acesyde** [comment]
Hi @pandaboy6621 can you describe a little more your need ? :)

**Comment by pandaboy6621** [comment]
I don’t have an icon color picker for my media player card

**Comment by AbeltjeNL** [comment]
Same here, it would be nice to be able to change the icon in the media player card. Also in yaml mode, it's not an option.

**Comment by Christophe999s** [comment]
Agreed, it would be nice to have an option to change the color, just like with the entity card

**Comment by rk9qn3j** [comment]
Yeah, I find it strange that the default color is indigo. It should by default follow the default color scheme, with the option to choose another color.

**Comment by rk9qn3j** [comment]
Any update on this?
